### DISCUSSION TITLE
Concurrent PAI + Paradox Insite Gold app

### DISCUSSION THREAD

**Discussion by gfplab-scaffardi** [ParadoxAlarmInterface/pai#483] [General]
Hi! Is it possible to use both together? 

I'd like to monitor my SP6000+IP150 (fw 5.2.19 with bootloader 2.14.000) using Home Assistant but handling all this stuff remotely with Paradox app, that is simpler than Home Assistant.

I read about a special firmware that allows the IP module concurrent connections of the app and hass (https://github.com/ParadoxAlarmInterface/pai/issues/356) but it's not applicable to me (that version of infield does not upgrade my IP150 bootloader version).

Any ideas? 
How do you arm/disarm/monitor the alarm only with home assistant?!?

**Comment by yozik04** [comment]
It is not possible to use PAI and Paradox app simultaniously. PAI requires constant connection to the panel that does not allow any other parallel connections. You can use HomeAssistant to arm and disarm, bypass zones, get alarm trigger notifications and many more.

**Reply by gfplab-scaffardi** [reply]
OMG :-o Maybe i misunderstood that link... a solution could be creating an intermediate proxy between IP150:10000 port and HASS/Insite Gold. I have to study that.

But what do you use in HASS to handle alarms? Alarmo seems to break itself while upgrading Home Assistant. It's not stable enought in my opinion.

**Reply by gfplab-scaffardi** [reply]
> It is not possible to use PAI and Paradox app simultaniously. PAI requires constant connection to the panel that does not allow any other parallel connections. You can use HomeAssistant to arm and disarm, bypass zones, get alarm trigger notifications and many more.

And what about using the 4-wire combus/communication bus that is used to connect Paradox accessories? I see many projects able to connect and Arduino board to it to read/write data to the Paradox Alarm. I could be these kind of adapters but i dont know if PAI is able to work with it because there's no documentation about this 4-wire protocol in this project. I am a programmer and i'm surprised nobody wrote a middleware between PAI and this combus interface of Paradox.

**Reply by yozik04** [reply]
PAI does not understand combus protocol. We use BabyWare protocol.
